### PR TITLE
fix somatic tasks and add supporting tasks

### PR DESCRIPTION
**Pull request by DavyCats**

### CODE REVIEW

File: manta.wdl [hunk 1]
```diff
@@ -0,0 +1,63 @@
+version 1.0
+
+task Somatic {
```
Comment (ffinfo) [inline manta.wdl]: Why is task called Somatic while the tool is called manta?

File: samplesheet.wdl [hunk 2]
```diff
@@ -1,4 +1,5 @@
 version 1.0
+
 struct Readgroup {
```
Comment (ffinfo) [inline samplesheet.wdl]: This complete file should not be in tasks. This can be pipeline specific

Comment (DavyCats) [reply samplesheet.wdl]: @rhpvorderman placed it here, the idea being that this is a basic version that could easily be reused. If a pipeline needs something special, they don't have to use this one. I'm not too opinionated on whether this should be here or not, but maybe Ruben has something to say about this?

File: manta.wdl (hunk 1 above)
Comment (DavyCats) [reply manta.wdl]: Manta has two possible configurations, somatic and germline. I didn't implement the option for germline SV calling, but as that might be implemented in the future I thought naming this somatic would be more appropriate (like how the BWA mapping task is called 'Mem' and not 'Bwa').

File: samplesheet.wdl (hunk 2 above)
Comment (ffinfo) [reply samplesheet.wdl]: Is this not why we have pipeline-template?

File: manta.wdl (hunk 1 above)
Comment (ffinfo) [reply manta.wdl]: :+1:

**Pull request merged**